src/main/java/mate.academy/lib/Inject.java

```diff
@@ -1,5 +1,9 @@
 package mate.academy.lib;
 
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
```

Comment:
You need to add @Target annotation, according to [mistakes](https://github.com/mate-academy/jv-dependency-injection/blob/main/checklist.md).